lib/Cake/Console/ConsoleErrorHandler.php

```diff
@@ -60,7 +60,7 @@ public function handleException($exception) {
 		));
 		$code = $exception->getCode();
 		$code = ($code && is_int($code)) ? $code : 1;
-		return $this->_stop($code);
+		$this->_stop($code);
```

Comment:
I think we should keep the current behavior as it is, when we found this kind of code. The current behavior allow people to return any value by overriding `_stop()`. So this change might break exiting applications. Instead, it would be better to change phpDoc blocks only.

Reply:
These returns are needed for mocks to work properly.

Reply:
Yes. Must be reverted and the doc blocks fixed instead.

Reply:
Reverted. 
I do not know the optimal answer for doc blocks :,-(